src/main/java/io/ebeaninternal/api/SpiTransactionProxy.java

```diff
@@ -325,7 +325,7 @@ public Object getUserObject(String name) {
 
   @Override
   public void depth(int diff) {
-    transaction.depth();
+    transaction.depth(diff);
```

Comment:
I think this is an error, that I have found while debugging

Reply:
Logged as: https://github.com/ebean-orm/ebean/issues/1705